src/snowflake/cli/_plugins/init/commands.py

```diff
@@ -203,6 +203,9 @@ def init(
     is_remote = any(
         template_source.startswith(prefix) for prefix in ["git@", "http://", "https://"]  # type: ignore
     )
+    args_error_msg = (
+        "Check whether --template and --template_source arguments are correct."
+    )
```

Comment:
Why is it created in command and passed to function as argument?

Reply:
I wanted to keep it close to actual arguments definition - in case their names will change, no one will remember to edit an error message in function 150 lines above

Reply:
So maybe something like:
```
    args_error_msg = (
        "Check whether {TemplateOption.param_decls} and {SourceOption.param_decls} arguments are correct."
    )
```
This should update the message automagically

Reply:
Neat!
Changed